apps/web/src/Pages/Airdrop/Table/Table.tsx

```
@@ -162,268 +177,302 @@ const AirdropTable: React.FC<LockupProps> = ({
+                    />
+                  </TableCell>
+                  <TableCell>
+                    {humanizeDuration(
+                      Number(contract.global_distribution_count) *
+                        Number(contract.global_distribution_seconds) *
+                        1000,
+                      { units: ["y"], round: true }
+                    )}
+                  </TableCell>
+                  <TableCell>
+                    {!!selection ? (
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)